src/table/header/table-header.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component, HostBinding } from "@angular/core";
+
+@Component({
+	selector: "ibm-table-header",
+	template: `
+		<ng-content selector="ibmTableHeaderTitle"></ng-content>
+		<ng-content selector="ibmTableHeaderDescription"></ng-content>
```

Comment:
This can just be one `ng-content` with no selector since we're not doing anything special with ordering 👍